docs/disaster-recovery.md

```diff
@@ -0,0 +1,443 @@
+
+:memo: As mentioned in the pre-requisites, periodic or one time backup of
+ the application should be available for restore on the secondary site (cluster-b).
+
+Follow the below steps for planned migration of workload from primary
+ cluster to secondary cluster:
+
+* Scale down all the application pods which are using the
+ mirrored PVC on the Primary Cluster
+* Take a back up of PVC and PV object from the primary cluster.
+ This can be done using some backup tools like
+ [velero](https://velero.io/docs/main/).
+* Update `replicationState` to `secondary` in VolumeReplication CR at Primary Site.
+ When the operator sees this change, it will pass the information down to the
+  driver via GRPC request to mark the dataSource as `secondary`.
+* If you are manually recreating the PVC and PV on the secondary cluster,
+ remove the `claimRef` section in the PV objects.
+* Recreate the storageclass, PVC, and PV objects on the secondary site.
+* As you are creating the static binding between PVC and PV, a new PV won’t
+ be created here, the PVC will get bind to the existing PV.
+* Create the VolumeReplicationClass on the secondary site.
+* Create the VolumeReplications for all the PVC’s for which mirroring
+ is enabled
+  * `replicationState` should be `primary` for all the PVC’s on
+   the secondary site.
+* Check whether the image is marked `primary` on the secondary site
+ by verifying it in VolumeReplication CR status.
+* Once the Image is marked as `primary`, the PVC is now ready
+ to be used. Now, we can scale up the applications to use the PVC.
+
+### Failback
+
+To perform a failback operation to primary cluster in case of planned migration
+, just repeat the Failback steps in vice-versa.
+
+>:memo: **Remember**: We already have all the required yamls to bring up the
+> resources in the primary cluster; so we can skip the backup-restore operations
+> in case of failback.
+
+## Disaster Recovery
+
+> Use cases: Natural disasters, Power failures, System failures, and crashes, etc.
+
+### Failover (abrupt shutdown)
+
+In case of Disaster recovery, create VolumeReplication CR at the Secondary Site.
+ Since the connection to the Primary Site is lost, the operator automatically
+  sends a GRPC request down to the driver to forcefully mark the dataSource as `primary`.
+
+* If you are manually creating the PVC and PV on the secondary cluster, remove
+ the claimRef section in the PV objects.
+* Create the storageclass, PVC, and PV objects on the secondary site.
+* As you are creating the static binding between PVC and PV, a new PV won’t be
+ created here, the PVC will get bind to the existing PV.
+* Create the VolumeReplicationClass and VolumeReplication CR on the secondary site.
+* Check whether the image is `primary` on secondary site, by looking at toolbox pod
+ or VolumeReplication CR status.
+* Once the Image is marked as `primary`, the PVC is now ready to be used. Now,
+ we can scale up the applications to use the PVC.
+
```

Comment:
add a note that as its Async DR you don't get the complete data you will get crash-consistent data based on the snapshot interval time.